src/main/webapp/js/common.js

```diff
@@ -138,10 +138,27 @@ var NAME_MAX_LENGTH = 40;
 var INSTITUTION_MAX_LENGTH = 64;
 
 $(document).on('ajaxComplete ready', function() {
-    $('[data-toggle="tooltip"]').tooltip({html: true, container: 'body'});
+    /**
+     * Initializing then disabling is better than simply
+     * not initializing for mobile due to some tooltips-specific
+     * code that throws errors.
+    */
+    var $tooltips = $('[data-toggle="tooltip"]');
+    $tooltips.tooltip({html: true, container: 'body'});
+    if (isTouchDevice()) {
+        $tooltips.tooltip('disable');
+    }
 });
 
 /**
+ * Checks if the current device is touch based device
+ * Reference: https://github.com/Modernizr/Modernizr/blob/master/feature-detects/touchevents.js
+ */
+function isTouchDevice() {
+    return true === (('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch);
```

Comment:
I understand that JS precedence rules ensure && is evaluated before ||, but please add brackets for readability sake :-)